bor/keeper.go

```diff
@@ -164,6 +159,7 @@ func (k *Keeper) GetSpanList(ctx sdk.Context, page uint64, limit uint64) ([]hmTy
 	iterator := hmTypes.KVStorePrefixIteratorPaginated(store, SpanPrefixKey, uint(page), uint(limit))
 
 	// loop through validators to get valid validators
+	var spans []hmTypes.Span
```

Comment:
could we pre-allocate capacity of len(iterator) for the slice?

Reply:
we don't know the len here

Reply:
agree - https://github.com/cosmos/cosmos-sdk/blob/acbf0dfee61f2cefd55d4529b203180c5c6c5224/orm/model/ormtable/iterator.go#L20